src/vs/base/parts/tree/browser/treeModel.ts

```diff
@@ -513,6 +514,7 @@ export class Item {
 						staleItems[staleItemId].dispose();
 					}
 				}
+
```

Comment:
blank line can be removed

Reply:
The styling of the remaining code file has lines there so keeping it for now.